request_a_govuk_domain/request/admin/model_admins.py

```diff
@@ -407,14 +413,17 @@ def save_model(self, request, obj, form, change):
             obj.application.status = ApplicationStatus.IN_PROGRESS
         # Change the owner to be the current user regardless if there is already a user
         # assigned or not
+        LOGGER.info(
+            f"Application {obj.application.reference} changed by {obj.application.owner}"
```

Comment:
Sorry :(, this line has to come after the obj.application.owner = request.user setting or use request.user as the parameter value